app/controllers/users_controller.rb

```diff
@@ -0,0 +1,28 @@
+class UsersController < ApplicationController
+  before_action :set_user
+
+  def account
```

Comment:
What's the reasoning behind using an `account` method here instead of `show` to conform to normal CRUD operations?  I get the desire in the route structure, but the route can be decoupled from the controller method name.

Reply:
Not such reason other than being explicit in the method name for functionality. I can change it to show to conform to normal CRUD operations.